src/widgets/DataTable/TouchableCell.js

```diff
@@ -68,7 +68,7 @@ TouchableCell.propTypes = {
   onPressAction: PropTypes.func.isRequired,
   dispatch: PropTypes.func.isRequired,
   renderChildren: PropTypes.func.isRequired,
-  TouchableComponent: PropTypes.func.isRequired,
+  TouchableComponent: PropTypes.node,
```

Comment:
node or element? A 30sec research tells me that a `number` would be valid here, which would be poop.

Reply:
Yeah, so I went a little further down the rabbit hole. All kinds of answers and I can't find a solid standard. I've had the most 'success' with `node` but happy to change to anything